website/src/pages/index.js

```diff
@@ -8,36 +8,6 @@ import styles from './styles.module.css';
 
 const features = [
   {
```

Comment:
It looks strange with no content here. When we have multiple cheatsheets we should link to them from here, but for now I propose we keep the quick links to the v1 pages. It just looks unfinished to have nothing here.

Reply:
added back the old content, modifying to new paths to the docs - updating this if needed can happen in a following PR